pkgs/servers/jellyfin/web.nix

```diff
@@ -1,41 +1,44 @@
-{
-  lib,
-  stdenv,
-  overrideSDK,
-  fetchFromGitHub,
-  buildNpmPackage,
-  nix-update-script,
-  pkg-config,
-  xcbuild,
-  pango,
-  giflib,
-  darwin,
+{ lib
+, stdenv
+, overrideSDK
+, fetchFromGitHub
+, buildNpmPackage
+, jellyfin
+, nix-update-script
+, pkg-config
+, xcbuild
+, pango
+, giflib
+, darwin
+,
 }:
 let
   # node-canvas builds code that requires aligned_alloc,
   # which on Darwin requires at least the 10.15 SDK
   stdenv' =
     if stdenv.isDarwin then
-      overrideSDK stdenv {
-        darwinMinVersion = "10.15";
-        darwinSdkVersion = "11.0";
-      }
+      overrideSDK stdenv
+        {
+          darwinMinVersion = "10.15";
+          darwinSdkVersion = "11.0";
+        }
```

Comment:
Enlarges the diff for no good reason.

Reply:
Fixed with `nixfmt-rfc-style`.

Reply:
> Fixed with `nixfmt-rfc-style`.

I'm not sure what tool are you talking about, (is it https://github.com/NixOS/nixfmt ? ) In either case, it's OK to use it if it is official, but I'd recommend apply it's changes in a dedicated commit. Also, I would recommend using 3 different commits for the updates, like I pushed to this PR. Together with the formatting commit, it will become:

```
jellyfin: 10.9.1 -> 10.9.3
jellyfin-web: use version from jellyfin, to avoid version mismatch
jellyfin-web: 10.9.2 -> 10.9.3
jellyfin: format with NixOS/nixfmt
jellyfin-web: format with NixOS/nixfmt
```

Reply:
I rebased to use three commits; the first two for the updates of `jellyfin` and `jellyfin-web`, respectively, and the third for adding the `assert` to `jellyfin-web`. I don't think it's idiomatic of a `nixpkgs` PR to make formatting a separate commit unless the *only* change to the `.nix` file is that it's being formatted.

Reply:
> I don't think it's idiomatic of a `nixpkgs` PR to make formatting a separate commit unless the _only_ change to the `.nix` file is that it's being formatted.

OK, not a big deal. It'd still be nice if you'd share (ideally in the commit message text), exactly what tool did you use to format the expression, I'm still not sure which one it is.

Reply:
> OK, not a big deal. It'd still be nice if you'd share (ideally in the commit message text), exactly what tool did you use to format the expression, I'm still not sure which one it is.

In the outdated version of my branch, I was using `nixpkgs-fmt`. I'm now using `nixfmt-rfc-style`, which is the package name for the current, [`RFC 166`](https://github.com/NixOS/nixfmt/issues/153)-compliant, version of `nixfmt`. I edited the PR message to explain that I formatted `jellyfin`'s `default.nix`, and that `jellyfin-web`'s `web.nix` was already formatted in this manner, so it didn't need to be formatted.

Reply:
Just edited the [commit message](https://github.com/NixOS/nixpkgs/pull/314775/commits/a7441feb23f2ba370fb3ae130f51113c5c72b8f2) for `jellyfin`'s update to indicate that it was formatted.

Reply:
OK I wasn't aware of this package, thanks for cooperating :) Will merge when CI is green.